modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IIdentityRoleRepository.cs

```diff
@@ -21,5 +21,9 @@ public interface IIdentityRoleRepository : IBasicRepository<IdentityRole, Guid>
             bool includeDetails = false,
             CancellationToken cancellationToken = default
         );
+
+        Task<List<IdentityRole>> GetDefaultOnesAsync(
+            CancellationToken cancellationToken = default
```

Comment:
Add `bool includeDetails = false` parameter to be compatible to other methods. See https://docs.abp.io/en/abp/latest/Best-Practices/Repositories to learn the rules. The section below explains this rule:

![image](https://user-images.githubusercontent.com/1210527/73735444-4f343a00-4750-11ea-9422-a9f7ba7249b9.png)